torch/lib/c10d/TCPStore.hpp

```diff
@@ -86,6 +104,8 @@ class TCPStore : public Store {
 
   bool deleteKey(const std::string& key) override;
 
+  void watchKey(const std::string& key, CallbackFunction callback);
```

Comment:
We probably need some docs for this function somewhere in terms of when CallbackFunction is called and with what args. Also, don't we need to define this in Store.hpp and override it here?

Reply:
Thanks, I will make these changes